pkg/crawler/crawler.go

```diff
@@ -262,3 +275,36 @@ func (c *Crawler) fetchSHA1(url string) ([]byte, error) {
 	}
 	return sha1b, nil
 }
+
+func (c *Crawler) fetchPOMLicense(url string) (string, error) {
+	resp, err := c.http.Get(url)
+	if resp.StatusCode == http.StatusNotFound {
+		return "", nil
+	}
+	if err != nil {
+		return "", xerrors.Errorf("can't get pom xml from %s: %w", url, err)
+	}
+	defer resp.Body.Close()
+
+	var pomProject gopom.Project
```

Comment:
What if we just add Structs for pom files(i mean `Licenses` and  `Name`) in https://github.com/aquasecurity/trivy-java-db/blob/main/pkg/crawler/types.go?

Reply:
Makes sense. Updated the PR

Reply:
Looks like you need to run `go mod tidy`

Reply:
Done